test/UnitTests/Microsoft.OData.Edm.Tests/Csdl/EdmValueParserTests.cs

```diff
@@ -433,52 +426,46 @@ public void TryParseFloatThatOverFlowsShouldBeFalse()
         }
 #endregion Decimal
 
-#region TimeOfDay
+#region TimeOnly
         [Fact]
         public void ParseTimeOfDayNullShouldThrowFormatException()
         {
-            TimeOfDay? result;
-            Assert.False(EdmValueParser.TryParseTimeOfDay(null, out result));
+            TimeOnly? result;
+            Assert.False(EdmValueParser.TryParseTimeOnly(null, out result));
         }
 
         [Fact]
         public void ParseTimeOfDayWithEmptyStringShouldThrowFormatException()
         {
-            TimeOfDay? result;
-            Assert.False(EdmValueParser.TryParseTimeOfDay(string.Empty, out result));
+            TimeOnly? result;
+            Assert.False(EdmValueParser.TryParseTimeOnly(string.Empty, out result));
         }
 
         [Fact]
         public void ParseTimeOfDayWithSpaceShouldThrowFormatException()
         {
-            TimeOfDay? result;
-            Assert.False(EdmValueParser.TryParseTimeOfDay(" ", out result));
+            TimeOnly? result;
+            Assert.False(EdmValueParser.TryParseTimeOnly(" ", out result));
         }
 
-        [Fact]
-        public void ParseTimeOfDayWithInvalidParameterShouldThrowFormatException()
+        [Theory]
+        [InlineData("-1:0:0.0")]
+        [InlineData("0:60:59.1")]
+        [InlineData("0:59:60.2")]
+        public void ParseTimeOfDayWithInvalidParameterShouldThrowFormatException(string invalidTime)
         {
-            var invalidTimeOfDays = new[]
-            { 
-                 "-1:0:0.0",
-                 "0:60:59.1",
-                 "0:59:60.2",
-                 "0:30:14.10000000",
-            };
-
-            foreach (var invalidTimeOfDay in invalidTimeOfDays)
-            {
-                TimeOfDay? result;
-                Assert.False(EdmValueParser.TryParseTimeOfDay(invalidTimeOfDay, out result));
-            }
+            TimeOnly? result;
+            Assert.False(EdmValueParser.TryParseTimeOnly(invalidTime, out result));
         }
 
-        [Fact]
-        public void TryParseTimeOfDayWithValidParameterShouldParseCorrectly()
+        [Theory]
+        [InlineData("0:30:14.10000000", "00:30:14.1000000")]
+        [InlineData("1:12:5.009000", "01:12:05.0090000")]
+        public void TryParseTimeOfDayWithValidParameterShouldParseCorrectly(string validTime, string expectedODataString)

```

Comment:
I see quite a few "TimeOfDay" references in the tests. Maybe you can do a find-and-replace to replace them with "TimeOnly"